dedupe/labeler.py

```diff
@@ -278,6 +282,27 @@ def _index_predicates(self, candidates):
         for pred in blocker.index_predicates:
             pred.freeze(records)
 
+    def _candidates(self, data, sample_size):
+
+        weights = {}
+        for predicate, covered in self.block_learner.comparison_cover.items():
+            # each predicate gets to vote for every record pair it covers. the
+            # strength of that vote is in inverse proportion to the number of
+            # records the predicate covers.
+            #
+            # if a predicate only covers a few record pairs, the value of
+            # the vote it puts on those few pairs will be worth more than
+            # a predicate that covers almost all the record pairs
+            weight = 1 / len(covered)
```

Comment:
this is the first pass at a weighting scheme. it seems pretty reasonable to start with.